docs/concepts/crds/restic.md

```diff
@@ -78,22 +78,22 @@ The default value for `spec.type` is `online`. For offline backup you need to sp
 | `keepWeekly`  | integer | --keep-weekly n    | For the last n weeks which have one or more snapshots, only keep the last one for that week.       |
 | `keepMonthly` | integer | --keep-monthly n   | For the last n months which have one or more snapshots, only keep the last one for that month.     |
 | `keepYearly`  | integer | --keep-yearly n    | For the last n years which have one or more snapshots, only keep the last one for that year.       |
-| `keepTags`    | array   | --keep-tag <tag>   | Keep all snapshots which have all tags specified by this option (can be specified multiple times). [`--tag foo,tag bar`](https://github.com/restic/restic/blob/master/doc/060_forget.rst) style tagging is not supported. |
+| `keepTags`    | array   | --keep-tag <tag>   | Keep all snapshots which have all tags specified by this option (can be specified multiple times). [`--tag foo,tag bar`](https://github.com/backup/backup/blob/master/doc/060_forget.rst) style tagging is not supported. |
```

Comment:
Use: https://github.com/restic/restic/blob/master/doc/060_forget.rst  instead of https://github.com/backup/backup/blob/master/doc/060_forget.rst